<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrajectoryVisualizer</class>
 <widget class="QMainWindow" name="TrajectoryVisualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrajectoryVisualizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QFrame" name="graphFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>550</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="Graph" name="graph">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>531</width>
         <height>411</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
      </widget>
      <widget class="QSlider" name="startTimeSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>430</y>
         <width>411</width>
         <height>29</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="endTimeSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>460</y>
         <width>411</width>
         <height>29</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>500</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Time Range Fixed</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="startTimeEdit">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>430</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="endTimeEdit">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>460</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>99</string>
       </property>
      </widget>
      <zorder>graph</zorder>
      <zorder>startTimeSlider</zorder>
      <zorder>endTimeSlider</zorder>
      <zorder>checkBox</zorder>
      <zorder>startTimeEdit</zorder>
      <zorder>endTimeEdit</zorder>
      <zorder>pointInfoFrame</zorder>
      <zorder>pointInfoFrame</zorder>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="pointInfoFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="trajectoryLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Trajectory</string>
       </property>
      </widget>
      <widget class="QLabel" name="positionLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>71</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
      <widget class="QLabel" name="velocityLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Velocity</string>
       </property>
      </widget>
      <widget class="QLabel" name="pointInfoTitle">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Point Information</string>
       </property>
      </widget>
      <widget class="QLabel" name="colorLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
      <widget class="QLabel" name="timeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="trajectoryEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="velocityEditX">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="positionEditY">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="positionEditX">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="velocityEditY">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="colorEditH">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>190</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="colorEditS">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>190</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="colorEditV">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>190</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="timeEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>281</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>281</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <zorder>trajectoryLabel</zorder>
      <zorder>positionLabel</zorder>
      <zorder>velocityLabel</zorder>
      <zorder>pointInfoTitle</zorder>
      <zorder>colorLabel</zorder>
      <zorder>timeLabel</zorder>
      <zorder>graphFrame</zorder>
      <zorder>trajectoryEdit</zorder>
      <zorder>velocityEditX</zorder>
      <zorder>positionEditY</zorder>
      <zorder>positionEditX</zorder>
      <zorder>velocityEditY</zorder>
      <zorder>colorEditH</zorder>
      <zorder>colorEditS</zorder>
      <zorder>colorEditV</zorder>
      <zorder>timeEdit</zorder>
      <zorder>line</zorder>
      <zorder>line_2</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelect_Point"/>
    <addaction name="actionAdd_Point"/>
    <addaction name="actionDelete_Point"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionSelect_Point">
   <property name="text">
    <string>Select Point</string>
   </property>
  </action>
  <action name="actionAdd_Point">
   <property name="text">
    <string>Add Point</string>
   </property>
  </action>
  <action name="actionDelete_Point">
   <property name="text">
    <string>Delete Point</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Graph</class>
   <extends>QGraphicsView</extends>
   <header>graph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
